### PR TITLE
NICPS-197 : Login History - Front End Splash Page and Additional Columns

### PR DESCRIPTION
**Pull request by kvc2983**
Changes for showing login history on splash page

### CODE REVIEW

File: WebRoot/js/zimbraMail/core/ZmZimbraMail.js [hunk 1]
```diff
@@ -281,7 +281,8 @@ ZmZimbraMail.killSplash =
 function() {
 	// 	Splash screen is now a part of the skin, loaded in statically via the JSP
 	//	as a well-known ID.  To hide the splash screen, just hide that div.
-	Dwt.hide("skin_container_splash_screen");
+	console.log("IN KILL SPLASH");
```
Comment (k-kato) [inline WebRoot/js/zimbraMail/core/ZmZimbraMail.js]: Is the log necessary for release?

File: WebRoot/skins/_base/base3/splash.html [hunk 2]
```diff
@@ -20,42 +20,91 @@
...
-            } else {
-              document.write(ZmMsg.splashScreenCopyright);
-            }
-          </script>
-        </div>
-    </div>
-	<div class="decor2"></div>
-</div>
-<!-- END SPLASH SCREEN -->
+<!-- BEGIN SPLASH SCREEN -->
```
Comment (k-kato) [inline WebRoot/skins/_base/base3/splash.html]: When I see code diff on github, most of the code is treated as 'changed'. Has the line break type (LF, CR, CR+LF) been changed? It should not be changed typically.

File: WebRoot/js/zimbraMail/core/ZmZimbraMail.js (hunk 1 above)
Comment (kvc2983) [reply WebRoot/js/zimbraMail/core/ZmZimbraMail.js]: Removed Console log statement.

File: WebRoot/skins/_base/base3/splash.html (hunk 2 above)
Comment (kvc2983) [reply WebRoot/skins/_base/base3/splash.html]: Fixed

File: WebRoot/skins/_base/base3/splash.html [hunk 3]
```diff
@@ -40,6 +40,54 @@ <h1><div id='ZLoginBannerImage' class='ImgLoginBanner' onclick='showCompanyUrl()
 				</div>
 				<div class="switch" id="splashScreenSwitchContainer">
 					<script>if(!window.isOffline) { document.write(ZmMsg.splashScreenSwitch) }</script>
+				</div>
+				<div class="container-table100">
+							<div class="wrap-table100">
```
Comment (k-kato) [inline WebRoot/skins/_base/base3/splash.html]: nit: decrease indent

File: WebRoot/skins/_base/base3/splash.html [hunk 4]
```diff
@@ -40,6 +40,54 @@ <h1><div id='ZLoginBannerImage' class='ImgLoginBanner' onclick='showCompanyUrl()
...
+										<thead>
+											<tr class="table100-head">
+												<th class="column1"><script>document.write(ZmMsg.zimbraLoginHistorySrNo)</script></th>
+												<th class="column2"><script>document.write(ZmMsg.zimbraLoginHistoryGeoLocationIP)</script></th>
+												<th class="column3"><script>document.write(ZmMsg.zimbraLoginHistoryTimestamp)</script></th>
+												<th class="column4"><script>document.write(ZmMsg.zimbraLoginHistoryProtocol)</script></th>
+											</tr>
+										</thead>
+										<tbody class="table_body">
+												<tr>
```
Comment (k-kato) [inline WebRoot/skins/_base/base3/splash.html]: nit: decrease indent

Comment (kvc2983) [reply WebRoot/skins/_base/base3/splash.html]: Fixed

**Pull request merged**